### PR TITLE
[DEV-3407] bankFee for chargebacks

### PR DESCRIPTION
**Pull request by Yannick1712**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations (on [DFX services](https://dev.services.dfx.swiss))
  - Login/logout
  - Buy/sell payment request
  - KYC page

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

**Comment by Yannick1712** [comment]
- [ ] update services

File: src/subdomains/core/history/controllers/transaction.controller.ts [hunk 1]
```diff
@@ -310,11 +310,15 @@ export class TransactionController {
     if (transaction.targetEntity?.cryptoInput?.txType === PayInType.PAYMENT)
       throw new BadRequestException('You cannot refund payment transactions');
 
-    const feeAmount = transaction.targetEntity.cryptoInput
+    const dfxFeeAmount = transaction.targetEntity.cryptoInput
       ? await this.feeService.getBlockchainFee(transaction.targetEntity.cryptoInput.asset, false)
       : 0;
```
Comment (davidleomay) [inline src/subdomains/core/history/controllers/transaction.controller.ts:315]: Shouldn't this be network fee instead of dfx?

Comment (Yannick1712) [reply src/subdomains/core/history/controllers/transaction.controller.ts:315]: Hehe yeah your are right 😁

**Review by davidleomay — COMMENTED** [review]
- [x] Document endpoints

**Review by davidleomay — APPROVED** [review]

**Pull request merged**